tests/acceptance/16_cf-serverd/add_access_constraints_fail.cf

```diff
@@ -0,0 +1,45 @@
+body common control
+{
+  inputs => { "../default.cf.sub" };
+  bundlesequence => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+  commands:
+    "/bin/echo -n '=line_number=' && /bin/grep -n GDB:.*add_access_constraints_fail.cf $(this.promise_dirname)/../../../cf-serverd/server_transform.c | cut -d: -f1"
+      contain => shell_command,
+      module => "true";
+
+  files:
+    "$(sys.workdir)/add_access_constraints_fail.gdb"
+      create => "true",
+      edit_template => "$(this.promise_dirname)/add_access_constraints_fail.gdb.mustache",
+      template_data => ' { "line_number": "$(echo.line_number)" } ',
+      template_method => "mustache";
+}
```

Comment:
Would be great to have this as a shared `method`.

Reply:
I think this is too complicated at this time. Pathing issues get weird when I pulled some of that policy into a bundle in dcs.cf.sub at tests/acceptance. I could find an "easy" way to pull things out and make them in a shared method. Maybe add a JIRA for it or just let it be?

Reply:
done: https://tracker.mender.io/browse/ENT-4125

Do you think we should create an epic for "Increase Code Coverage"? It is really just a standard ongoing maintenance task like "reduce technical debt".

Reply:
> Do you think we should create an epic for "Increase Code Coverage"?

Yes, please. It might be a nice "future workshop theme".